PIDs:
  X:
    kp: 0.1
    kd: 0.1
    ki: 0.1
    minOutput: -0.03
    maxOutput: 0.03
    integratorMin: -0.5
    integratorMax: 0.5
  Y:
    kp: 0.1
    kd: 0.1
    ki: 0.1
    minOutput: -0.03
    maxOutput: 0.03
    integratorMin: -0.5
    integratorMax: 0.5
  Z:
    kp: 1
    kd: 1
    ki: 1
    minOutput: -0.1
    maxOutput: 0.1
    integratorMin: -0.1
    integratorMax: 0.1
  Yaw:
    kp: 10.0
    kd: 1
    ki: 0.9
    minOutput: -1
    maxOutput: 1
    integratorMin: 0.6
    integratorMax: 0.6
